src/node_i18n.h

```diff
@@ -60,6 +65,75 @@ int32_t ToUnicode(MaybeStackBuffer<char>* buf,
                   const char* input,
                   size_t length);
 
+struct ConverterDeleter {
+  void operator()(UConverter* pointer) const { ucnv_close(pointer); }
+};
+using ConverterPointer = std::unique_ptr<UConverter, ConverterDeleter>;
+
+class Converter {
+ public:
+  explicit Converter(const char* name, const char* sub = nullptr);
+  explicit Converter(UConverter* converter, const char* sub = nullptr);
+
+  UConverter* conv() const { return conv_.get(); }
+
+  size_t max_char_size() const;
+  size_t min_char_size() const;
+  void reset();
+  void set_subst_chars(const char* sub = nullptr);
+
+ private:
+  ConverterPointer conv_;
+};
+
+class ConverterObject : public BaseObject, Converter {
+ public:
+  enum ConverterFlags {
+    CONVERTER_FLAGS_FLUSH      = 0x1,
+    CONVERTER_FLAGS_FATAL      = 0x2,
+    CONVERTER_FLAGS_IGNORE_BOM = 0x4,
+    CONVERTER_FLAGS_UNICODE    = 0x8,
+    CONVERTER_FLAGS_BOM_SEEN   = 0x10,
+  };
+
+  static void Create(const v8::FunctionCallbackInfo<v8::Value>& args);
+  static void Decode(const v8::FunctionCallbackInfo<v8::Value>& args);
+  static void Has(const v8::FunctionCallbackInfo<v8::Value>& args);
+
+  SET_NO_MEMORY_INFO()
```

Comment:
Not that it was tracked before but you can also track the size of the UConverter explicitly with `TrackFieldWithSize("conv_", sizeof(UConverter), "UConverter")` since it's own by this (or maybe the size can be more precise, but it's at least better than nothing)

Reply:
Interesting... when I do that I get...

```
C:\Users\jasne\Projects\node\src\node_i18n.cc(385,64): error C2027: use of undefined type 'UConverter' [C:\Users\jasne\Projects\node\libnode.vcxproj]
C:\Users\jasne\Projects\node\deps\icu-small\source\common\unicode\unistr.h(42): message : see declaration of 'UConverter' [C:\Users\jasne\Projects\node\libnode.vcxproj]
```

Something odd with the definition of `UConverter`. I'll investigate that add do the memory tracking in a separate PR rather than block on it in this one

Reply:
Yeah, I think that’s why we didn’t have that check before.